frontend/apps/studio/src/global.css

```diff
@@ -0,0 +1,3 @@
+body {
+    margin: 0;
+}
```

Comment:
Might be worth adding more reset styles here, like [this](https://elad2412.github.io/the-new-css-reset/).

Reply:
I believe we may want to use [MUI resets](https://mui.com/material-ui/react-css-baseline/), right?

Reply:
Oh nice, yes! Didn't know they had that 🙌